src/core/zowe/core_for_zowe_sdk/profile_manager.py

```diff
@@ -224,6 +274,14 @@ def load(
                 )  # Define profile name that will be merged from other layers
             profile_props = {**profile_loaded.data, **profile_props}
 
+            # Validating $schema property for all the layers
+            if cfg._location and config_type in ("Project Config", "Global Config"):
+                path_config_json = cfg._location + "/zowe.config.json"
+                self.validate_schema(cfg, path_config_json, opt_in)
+            elif cfg._location and config_type in ("Project User Config", "Global User Config"):
+                path_config_json = cfg._location + "/zowe.config.user.json"
+                self.validate_schema(cfg, path_config_json, opt_in)
```

Comment:
I don't think validating the schema needs to be specific to the ProfileManager class. We could probably move the `validate_schema` method into the ConfigFile class, since its an action that applies to any individual config file. @zFernand0 do you agree? 😋

Reply:
it seems that this was already addressed 😋